Add docs for `huey` integration (#8176)
getsentry/sentry-docs: src/platforms/python/common/integrations/huey/index.mdx

---
title: huey
description: "Learn how to import and use the huey integration."
---

The huey integration adds support for the
[huey task queue library](https://huey.readthedocs.io/en/latest/).

## Install

To get started, install `sentry-sdk` from PyPI.

```bash
pip install --upgrade sentry-sdk
```

## Configure

Add `HueyIntegration()` to your `integrations` list:

<SignInNote />

```python
import sentry_sdk
from sentry_sdk.integrations.huey import HueyIntegration

sentry_sdk.init(
    dsn="___PUBLIC_DSN___",
    integrations=[
        HueyIntegration(),
    ],
    # Set traces_sample_rate to 1.0 to capture 100%
    # of transactions for performance monitoring.
    traces_sample_rate=1.0,
)
```

## Verify

```python
from huey import SqliteHuey

sentry_sdk.init(...)  # same as above

huey = SqliteHuey(filename='demo.db')

@huey.task()
def add(a, b):
    return a + b

with sentry_sdk.start_transaction(name="testing_huey"):
    result = add(1, 2)
```

> Running this will make a new transaction called `testing_huey` appear in the
> Performance section of [sentry.io](https://sentry.io). Note that it might
> take a couple moments for it to show up.

Reviewer: ```suggestion
Running this will create a new transaction called `testing_huey` in the
Performance section of [sentry.io](https://sentry.io). It may
take a couple of moments for the transaction to show up.
```
